Lib/typing.py

```diff
@@ -1731,6 +1732,22 @@ def cast(typ, val):
     return val
 
 
+def assert_type(val, typ, /):
+    """Assert (to the type checker) that the value is of the given type.
+
+    When the type checker encounters a call to assert_type(), it
+    emits an error if the value is not of the specified type::
+
+        def greet(name: str) -> None:
+            assert_type(name, str)  # ok
+            assert_type(name, int)  # type checker error
+
+    At runtime this returns the first argument unchanged.
```

Comment:
(Ditto, RE my related comment in the RST documentation file)

Reply:
Done, thanks for the review!